<?xml version="1.0"?>
<launch>
  <node pkg="rosbag" type="play" name="play" output="screen" args="--clock -l $(find medicine_calendar)/bag/medicine_calendar.bag"/>
  <node name="identify_case" pkg="medicine_calendar" type="identify_case" output="screen">
    <param name="threshold_plane" value="0.007"/> <!--値が小さいほど平面の条件が小さくなる-->
    <param name="number_neighbors" value="20"/>
    <param name="seddev_multhresh" value="1.0"/>
  </node>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find medicine_calendar)/rviz/identify_case.rviz" required="true" output="screen" />
</launch>
